cli/shared/src/main/scala/com/lightbend/rp/reactivecli/annotations/Annotations.scala

```diff
@@ -36,6 +36,8 @@ case class Annotations(
   memory: Option[Long],
   cpu: Option[Double],
   endpoints: Map[String, Endpoint],
+  remotingEndpointName: Option[String],
```

Comment:
Is this being used? I did a quick search and it seems that this field is never read.

Reply:
I can remove that one probably.